leapp/logger/__init__.py

```diff
@@ -53,22 +54,29 @@ def configure_logger():
     global _logger
     if not _logger:
         path = os.getenv('LEAPP_LOGGER_CONFIG', '/etc/leapp/logger.conf')
-        log_format = '%(asctime)s.%(msecs)-3d %(levelname)-8s PID: %(process)d %(name)s: %(message)s'
-        log_date_format = '%Y-%m-%d %H:%M:%S'
+
         if path and os.path.isfile(path):
             logging.config.fileConfig(path)
         else:  # Fall back logging configuration
             logging.Formatter.converter = time.gmtime
+            log_format = '%(asctime)s.%(msecs)-3d %(levelname)-8s PID: %(process)d %(name)s: %(message)s'
+            log_date_format = '%Y-%m-%d %H:%M:%S'
+            logging.basicConfig(
+                level=logging.DEBUG,
+                format=log_format,
+                datefmt=log_date_format,
+                stream=sys.stderr,
+            )
             logging.getLogger('urllib3').setLevel(logging.WARN)
             handler = LeappAuditHandler()
             handler.setFormatter(logging.Formatter(fmt=log_format, datefmt=log_date_format))
             logging.getLogger('leapp').addHandler(handler)
+            logging.StreamHandler().setLevel(logging.ERROR)
 
-        if os.getenv('LEAPP_VERBOSE', '0') == '1':
-            logging.getLogger('').setLevel(logging.DEBUG if os.getenv('LEAPP_DEBUG', '0') == '1' else logging.INFO)
-            handler = logging.StreamHandler(sys.stderr)
-            handler.setFormatter(logging.Formatter(fmt=log_format, datefmt=log_date_format))
-            logging.getLogger('').addHandler(handler)
+        if is_verbose():
+            for handler in logging.getLogger().handlers:
```

Comment:
you set for all handlers the log level here, that means you do not respect the fact that the audit log still should contain DEBUG data as we discussed it on the cabal.